sdk/src/client/api/types.rs

```diff
@@ -95,16 +95,16 @@ pub struct SignedTransactionData {
 #[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct SignedTransactionDataDto {
-    /// Transaction essence
-    pub transaction_payload: TransactionPayloadDto,
+    /// Transaction payload
+    pub transaction_payload: SignedTransactionPayloadDto,
```

Comment:
Maybe also prefix with `signed_`?

Reply:
Sure

Reply:
Tbh, we're only in a `SignedTransactionData` named type, I'd just call the field payload, is that fine?

Reply:
Yeah I'll do that

Reply:
that was the alternative I wanted to suggest :smile: